src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.json

```diff
@@ -36,9 +37,9 @@
        "System.Threading.Thread": "4.0.0-beta-*",
        "System.Xml.ReaderWriter": "4.0.10",
        "System.Xml.XDocument": "4.0.10",
-       "xunit": "2.1.0-beta3-*",
```

Comment:
Do you know if the rc version still keeps wanting to add a .props file whenever you build/restore it from VS in one of our projects?